Editor/Scripts/MeshSyncServerInspector.cs

```diff
@@ -86,23 +153,58 @@ public void DrawServerSettings(MeshSyncServer t)
         }
     }
 
+    private void DrawInstanceSettings(MeshSyncServer t)
+    {
+        var style = EditorStyles.foldout;
+        style.fontStyle = FontStyle.Bold;
+        t.foldInstanceSettings = EditorGUILayout.Foldout(t.foldInstanceSettings, "Instances", true, style);
+        if (t.foldInstanceSettings)
+        {
+            t.InstanceHandling = (BaseMeshSync.InstanceHandlingType)EditorGUILayout.EnumPopup("Instance handling", t.InstanceHandling);
```

Comment:
I am using the test code to get test meshes, and when I change this InstanceHandling variable, my meshes are gone, even though they aren't geometry nodes.
Is this expected ?

![changing_instance_handling](https://user-images.githubusercontent.com/7379311/177310472-8a54fecf-1900-4c69-a2d8-ed018c59ec17.gif)

Reply:
Changing the instance handling method clears the objects and needs a full sync from blender to rebuild them